File: helloworld_web_jar/src/main/java/org/example/demo/controller/ErrorController.java
package org.example.demo.controller;

import org.noear.solon.annotation.Controller;
import org.noear.solon.annotation.Mapping;
import org.noear.solon.annotation.Produces;
import org.noear.solon.boot.web.MimeType;

@Controller
public class ErrorController {
    @Produces(MimeType.TEXT_HTML_VALUE)
    @Mapping("/error")
    public String error(){
        return "<html><body>出错了：（</body></html>";
    }
}

File: helloworld_web_jar_kt/src/main/kotlin/com/example/demo/controller/ErrorController.kt
package com.example.demo.controller

import org.noear.solon.annotation.Controller
import org.noear.solon.annotation.Mapping
import org.noear.solon.annotation.Produces
import org.noear.solon.boot.web.MimeType


@Controller
class ErrorController {
    @Produces(MimeType.TEXT_HTML_VALUE)
    @Mapping("/error")
    fun error(): String {
        return "<html><body>出错了：（</body></html>"
    }
}

